README.md

```diff
@@ -1,6 +1,9 @@
 # Collection for Event-Driven Ansible
 
-This collection contains event source plugins, event filters and example rulebooks to be used with [ansible-rulebook](https://ansible-rulebook.readthedocs.io/en/stable/).
+This collection contains event source plugins, event filters and example rulebooks to be used with [eda-server](https://github.com/ansible/eda-server).
+Eda-collection is intended to provide the initial resources for getting started with the [eda-server](https://github.com/ansible/eda-server). Other third-party integrations should be delegated to the collections developed by the ansible community.
+Some of the existing plugins in this collection that do not fit as well in the scope of the collection are just keep them there for backward compatibility. 
+They will be eventually deprecated at some point.
```

Comment:
Hi @fkuep, we really appreciate your contribution and effort towards clarification. The EDA team is currently drafting the official statement regarding these points.

cc @josephpisciotta

Reply:
Yes it was meant as conversation starter, thats why I was a little laze and did not fix all mistakes.

Reply:
@sc68cal Would that have helped You not to waste the effort?
@Alex-Izquierdo 
- Is there a list of welcomed integrations ?
- Will You have public meetings where we would ask beforehand ?
- Are You planning to look for new homes for the tb orphaned plugins?

Reply:
@fkuep I appreciate your willingness to generate discussion on this issue.

I had a video call with the maintainers of this repository a month ago where I raised the issues you are also raising. The commitment from their side was to provide clarity around what is the correct way to move forward with community contributed event sources.

My point has been that discoverability is critical to the success of EDA. Everyone looking to use EDA is going to look at this repository first for their preferred event source (rabbit, mqtt, etc etc) and when they don't find it here they're probably going to just move on and implement something themselves and not use Ansible at all.

From the Red Hat side, my recollection is that they are concerned about certifying all the code that is in this repository and providing support for community contributed event sources.

I understand this concern, being a former employee of Mirantis (and also IBM), so my suggestion was that there be a clear distinction between the parts that Red Hat wishes to certify and support, versus the community contributed ones.

This is usually done with a `contrib` namespace or directory, that signifies that these are contributed by the open source community.

I suggested something like `ansible.eda.contrib` or an equivalent Ansible galaxy collection that could contain those community contributed event sources, and have very good links in this repo's documentation to point to that collection, so that people don't just throw their hands up and walk away from EDA.

Reply:
One other suggestion from the Red Hat side was that event sources should be contributed to the existing collections for that component. In the case of the RabbitMQ event source, it should be contributed to https://github.com/ansible-collections/community.rabbitmq 

I am fine with this suggestion, provided that my event source is accepted in that repository and links in this repository are added so that people can find it.

Reply:
Hi @sc68cal ,
> I suggested something like `ansible.eda.contrib` or an equivalent Ansible galaxy collection that could contain those community contributed event sources, and have very good links in this repo's documentation to point to that collection, so that people don't just throw their hands up and walk away from EDA.

You already have a [mock-queue](https://github.com/ansible/event-driven-ansible/pull/183/files#diff-2e9b91d0553d029c42f64895dc50b2294956566363738478e47865a1fac48367R61) as suggested e.g [here](https://www.ansible.com/blog/creating-custom-event-driven-ansible-source-plugins).

I beleive the default awx execution environment includes community.general .
@Alex-Izquierdo who builds the _default decision environment_  Your collection is in ?
Could _"ansible.eda.contrib"_ go there provided test are good ?
Could a non-profit _"ansible.eda.contrib"_  collection integrate with their tools and push to quay.io so that beginners can use this ?

Reply:
I guess you mean a community collection equivalent to community.general, let's say community.eda. 
ansible.eda.contrib would mean a plugin inside this collection which would not solve the problem. The current image of ansible-rulebook in quay.io serves as decision environment and already includes ansible.eda collection. The addition of such a hypothetical collection could be discussed.

Reply:
@Alex-Izquierdo Do You say that You are building the image [here](https://github.com/ansible/ansible-rulebook/blob/main/Dockerfile)  and are willing to discuss inclusion of an community.eda collection?

@sc68cal is community.eda  a good name ?

Reply:
community.eda works for me

Reply:
@Alex-Izquierdo I guess now we would like You to 
* go to Your Team for us and secure us some intent to cooperate.
* Please bring us back some [love of collaboration](https://www.redhat.com/en/about/our-culture) from them.
* _Hurry - it is cold._